src/shared/components/query/gene_lists.ts

```diff
@@ -1,72 +0,0 @@
-const gene_lists = [{
```

Comment:
How do you handle the default on the frontend now? We would like everything to work also outside of `cbioportal` war, so that at one point we can really separate them. If gene lists are only used in the new frontend maybe better to only store the default here. If they are used in both places, might be good to store a default in both places. If we only store it in `cbioportal` repo, I don't see how we can make the frontend work standalone.